server/user.js

```diff
@@ -0,0 +1,25 @@
+Meteor.methods({
+    user_signup: function(data) {
+        Accounts.createUser({
```

Comment:
can u put a validation here. 
e.g if email and id and type is not present it will throw an exception